help/dns_verification/dns_verification.rst

```diff
@@ -0,0 +1,87 @@
+.. _doc_dnsverification:
+.. _Dashboard: https://mailsac.com/dashboard
+.. _Domains: https://mailsac.com/domains
+
+DNS Verification
+================
+
+DNS (Domain Name System) is at the core of the internet and email delivery.
+When configuring a :ref:`BYO Domain <section_byod_configuration>` the following
+records need to be configured:
+
+- 2 MX Records
+- 1 DKIM Record
+- 1 SPF Record
+
+MX Record Verification
+----------------------
+
+The values of the MX Records are the :ref:`same <section_email_hosting>` for all
+Private and Public Domains hosted by Mailsac. The Host will be unique per
+domain.
+
+======== ================= ==================
+**Type** **Host**           **Value**
+MX       dogtreats.msdc.co 1 in.mailsac.com
+MX       dogtreats.msdc.co 5 alt.mailsac.com
+======== ================= ==================
+
+To verify a domain's MX Records the command line tool `dig` can be used.
+In this example, the domain `dogtreats.msdc.co` is being used.
+
+.. code-block:: bash
+
+   dig MX dogtreats.msdc.co
+
+.. image:: dig_mx_record.png
+
+The values in the `ANSWER SECTION` show that the records are configured
+correctly because it corresponds to the value in the table above.
+
+DKIM Record Verification
+------------------------
+
+DKIM Records are unique per domain. The value of the required DKIM record can
+be found in the Dashboard_ under Domains_, then click "Manage" and choose the
+"DNS Setup" tab.
+
+======== ========================================== ========================
+**Type** **Host**                                   **Value**
+TXT      mailsacrelay._domainkey.dogtreats.msdc.co. v=DKIM1; k=rsa; p=MI....
```

Comment:
A user contacted me about a week and a half ago. He had entered exactly the host like this. But in his case, his domain provider (like most) does not want the `dogtreats.msdc.co` part, only `mailsacrelay._domainkey.dogtreats`. It took a lot of back and forth, and me guessing to run dig against effectively `dig mailsacrelay._domainkey.dogtreats.msdc.co.msdc.co` (yes) to even determine this issue.

Perhaps we need to call out that this should be the "end result" of entering the records. Showing examples of:

- configuring example.com
- configuring mail.example.com
- configuring example.co.uk
- configuring mail.example.co.uk